scripts/monopoly.coffee

```diff
@@ -176,6 +180,21 @@ module.exports = (robot) ->
     currentIndex = players[playerIndex].location
     updateThisProperty(board, players, currentIndex, buyerIndex)
 
+  getAccount = (name, accounts) ->
+    _.find accounts, (account) => account.name.toLowerCase() == name.toLowerCase()
+
+  updatePlayerAccount = (player, amount) ->
+    accounts = robot.brain.get 'monopolyAccounts'
+    account = getAccount(player, accounts)
+    if account
+      balance = parseInt account.balance
+      if balance > amount
+        balance -= amount
+        account.balance = balance
+        robot.brain.set 'monopolyAccounts', accounts
+        robot.messageRoom process.env.HUBOT_ROOM_MONOPOLY, "$#{amount} subtracted from #{account.name}."
```

Comment:
The message to the monopoly room may be redundant, since the previous line was "Dallas pays the bank X for ..."; that's why i suggested adding a payment message to the Admin room since I didn't expect to need another one for the players room. If you want to have this one in the players room, then the Admin one is redundant.

Reply:
Good point. I'll remove that line and keep the admin one